pom.xml

```diff
@@ -17,13 +17,13 @@
     <tycho.version>1.0.0</tycho.version>
     <manifest-location>META-INF</manifest-location>
     <cs-studio.version>4.5</cs-studio.version>
+    <csstudio.version>4.5.0-SNAPSHOT</csstudio.version>
```

Comment:
Now you've got `cs-studio.version` AND `csstudio.version`.

It seems like this was what was there before, but presumably we need to pick one.

Reply:
They are used for different things:
- cs-studio.version is used for the update links
- csstudio.version is used to set the version number of org.csstudio.platform.libs.epics
I don't know the history of why platform.lib.epics is included in m.o.b. or why it has a CSS version number.

Reply:
Well, they might be, but it's pretty confusing to have the names almost identical.

I realise that it might have been me that caused this originally.

Reply:
Resolving whether this property should exist is a separate issue to fixing the current inconsistency